src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,7 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  for (let i = 0; i < employees.length; i++) {
+    if (employeeName === employees[i].name) return employees[i].role;
+  }
```

Comment:
In effect youre trying to find a property of a single instance of an array here. You can do this using the built in find function:
```
  employees.find((employee) => employee.name === employeeName).role;
```